desktop/mac/privileged-helper.md

```diff
@@ -10,19 +10,17 @@ This page contains information about the permission requirements for running and
 
 ## Permission requirements
 
-While Docker Desktop does not generally require the user running or installing it to have `root` privileges, in the default use case it needs `root` access to be granted on the first run. The first time Docker Desktop is launched the user gets an admin prompt to grant permissions for a privileged helper service to be installed. For subsequent runs, no `root` privileges are required. 
+In the default set up flow, Docker Desktop for Mac does not require root privileges for installation but does require `root` access to be granted on the first run. The first time that Docker Desktop is launched the user gets an admin prompt to grant permissions for a privileged helper service to be installed. For subsequent runs, no `root` privileges are required. 
 
 The reason for this is that Docker Desktop needs to perform a limited set of privileged operations using the privileged helper process `com.docker.vmnetd`. This approach allows, following the principle of least privilege, `root` access to be used only for the operations for which it is absolutely necessary, while still being able to use Docker Desktop as an unprivileged user.
 
-From version 4.11, it will be possible to avoid running a privileged service in the background by using `com.docker.vmnetd` for setup during installation and disabling it at runtime. In this case the user will not be prompted on the first run. Administrators would be able to do that by using the `–-user` flag on the [install command](../install/mac-install.md#install-from-the-command-line) which would:
+From version 4.11 of Docker Desktop for Mac it will be possible to avoid running the privileged helper service in the background by using the `--user` flag on the [install command](../install/mac-install.md#install-from-the-command-line). This will result in `com.docker.vmnetd` being used for set up during installation and then disabled at runtime. In this case, the user will not be prompted to grant root privileges on the first run of Docker Desktop. Specifically, the `--user` flag will:
 - Uninstall the previous `com.docker.vmnetd` if present
 - Set up symlinks for the user
 - Ensure that `localhost` and `kubernetes.docker.internal` are present in `/etc/hosts`
 
-This approach will have certain limitations:
-- Docker Desktop would only be able to be run by one user account per machine, namely the one specified in the `-–user` flag.
-- Ports 1-79 would be blocked - the containers would run but the port won’t be exposed on the host.
-- Spindump diagnostics for fine grained CPU utilization would not be gathered.
+This approach will have the following limitation:
+- Ports 1-79 will be blocked - the containers will run but the ports won’t be exposed on the host.
```

Comment:
Whilst DD only being able to run 1 user account per machine is a technical limitation i'm not sure that users will actually care about this. Regarding CPU diagnostics, is this something that we're gathering ? If so, I'm similarly unsure whether users care will care about its absence. Do you think they're important to mention  for users ?

Reply:
Regarding the first one, in most cases they likely won't care, but I think it's nice for it to be documented in case they try and wonder why it doesn't work. The diagnostics thing I added for transparency but I guess it doesn't matter too much.

Reply:
I'd personally keep the first item. It's always best to cater to the 0.001% of users that might care just in case.